src/modules/ergoCubEmotions/ergoCubEmotions.h

```diff
@@ -53,13 +38,13 @@ class ErgoCubEmotions : public yarp::os::RFModule, public ergoCubEmotions_IDL {
         double getPeriod();
 
         bool setEmotion(const std::string& command);
+        std::vector<std::string> availableEmotions();

```

Comment:
Thrift can probably map the `list` as defined in the IDL file into `std::vector`.
At any rate, you may try to use `std::list` here, for better consistency.

Reply:
Since thrift maps `list` as `std::vector`, if I try to use here `std::list` instead of vector, it returns an error when compiling.

Reply:
Ok! Good to know.